src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final String dateFormat = "dd.MM.yyyy";
+        final String headMessage = "Report for period ";
+        final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(dateFormat);
```

Comment:
DateTimeFormatter should be constant:
private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");

> Please pay your attetion , i can't follow with rules of "constant naming requirements" because according to rules, i must use only capital letters with "_" separation, but maven (mvn clean packeage ) trhrow error "Abbreviation in name 'DATETIME_FORMATTER' must contain no more than '2' consecutive capital letters." The same problem with other local constants.

if you create constant correctly (using all necessary key words `private static final ` checkstyle chacks will pass)
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
ok